core/src/commonMain/resources/MR/base/strings.xml

```diff
@@ -4,6 +4,10 @@
     <!--suppress CheckTagEmptyBody -->
     <string name="blank" translatable="false"></string>
     <string name="login">Login</string>
+    <string name="share_app_title">Share App</string>
```

Comment:
Strings as per coding style should be as per the text itself e.g. share_app not share_app_title.